src/main/java/seedu/edrecord/model/person/Name.java

```diff
@@ -13,7 +13,7 @@
             "Names should only contain alphanumeric characters and spaces, and it should not be blank";
 
     /*
-     * The first character of the address must not be a whitespace,
+     * The first character of the edrecord must not be a whitespace,
```

Comment:
I think it should be "name" here, since it's a regex to verify a person's name validity.
i.e. _The first character of the **name** must not be a whitespace, 
(The original AB3 must have over-copied the comments from other parts too...)